llvm/include/llvm/Object/DXContainer.h

```diff
@@ -135,6 +145,9 @@ class RootSignature {
   uint32_t getRootParametersOffset() const { return RootParametersOffset; }
   uint32_t getNumStaticSamplers() const { return NumStaticSamplers; }
   uint32_t getStaticSamplersOffset() const { return StaticSamplersOffset; }
+  llvm::iterator_range<const RootParameter *> getParameters() const {
+    return llvm::make_range(Parameters.begin(), Parameters.end());
+  }
```

Comment:
Here, to match similar APIs, we should call this `params` (and we could also potentially include `param_begin` and `param_end` for completeness)